app/classifier/world_wide_telescope.cjsx

```diff
@@ -93,32 +109,40 @@ class Plate
       makeStarCoord(xRange[0].value, yRange[0].value, xAxisDec), makeStarCoord(xRange[1].value, yRange[0].value, xAxisDec),
       makeStarCoord(xRange[1].value, yRange[1].value, xAxisDec), makeStarCoord(xRange[0].value, yRange[1].value, xAxisDec)
     ]
+
   scale: ->
     [star1, star2] = [ @coordCorners[0], @coordCorners[2] ]
     [xy1, xy2] = [ @xyCorners[0], @xyCorners[2] ]
     averageDec = (star1.dec + star2.dec) / 2
     deltaRA = ((star2.ra - star1.ra) * Math.cos(averageDec * Math.PI/180)) * 3600
     deltaDec = (star2.dec - star1.dec) * 3600
-    angularSep = Math.sqrt(Math.pow(deltaRA,2) + Math.pow(deltaDec,2))
-    pixelSep = Math.sqrt(Math.pow(xy1.x - xy2.x,2) + Math.pow(xy1.y - xy2.y, 2))
+    angularSep = Math.sqrt(Math.pow(deltaRA, 2) + Math.pow(deltaDec, 2))
+    pixelSep = Math.sqrt(Math.pow(xy1.x - xy2.x, 2) + Math.pow(xy1.y - xy2.y, 2))
     angularSep / pixelSep
+
   centerCoords: ->
     center =
       x: @starChart.width / 2
       y: @starChart.height / 2
       ra: (@coordCorners[0].ra + @coordCorners[2].ra) / 2
       dec: (@coordCorners[0].dec + @coordCorners[2].dec) / 2
     center
+
   getCropUrl: ->
     url = @url.replace(/^https?\:\/\//i, "")
     # TODO: we need to account for the fact that the size of the image might be different than
     # the size that it is displayed
     "http://imgproc.zooniverse.org/crop/#{@starChart.width}/#{@starChart.height}/#{@starChart.x}/#{@starChart.y}?u=#{url}"
+
   computeRotation: ->
-    if @starChart.xAxis == Axis.RA || @starChart.xAxis == Axis.GLAT then 0 else 90
+    if @starChart.xAxis == Axis.RA || @starChart.xAxis == Axis.GLAT
+    then 0
```

Comment:
I believe this introduces a different behavior than the original code because of indenting rules?